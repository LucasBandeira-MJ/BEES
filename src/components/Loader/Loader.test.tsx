import renderer from 'react-test-renderer'

import { Loader } from '.'

describe("Loader component", () => {
    it("should match snapshot", () => {
        const snapshot = renderer
        .create(<Loader />)
        .toJSON()

        expect(snapshot).toMatchSnapshot()
    })
})