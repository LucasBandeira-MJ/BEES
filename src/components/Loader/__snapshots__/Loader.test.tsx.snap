// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should match snapshot 1`] = `
<div
  className="sc-bczRLJ AfBzu"
>
  <img
    alt="beer icon"
    src="images/beer.svg"
  />
</div>
`;
